Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,16 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = props => {
+    
+    return (
+    <div className = "headerRow">
+        <h1>Lambda School</h1>
+        <h2>@LambdaSchool</h2>
+        {new Date().toString()}
+       
```

Comment:
This `Date` would be floating content. Wrap it in a `p` tag just the way it is, and for better time and date formatting that isn't so Verbose you could use `Date.getDate()`